mindsdb/integrations/handlers/footballApi_handler/players_table.py

```diff
@@ -0,0 +1,138 @@
+from mindsdb.integrations.utilities.sql_utils import extract_comparison_conditions
+
+
+class PlayersTable(APITable):
+    def select(self, query: ast.Select) -> pd.DataFrame:
+        """Pulls data from the Football API
+
+                Parameters
+                ----------
+                query : ast.Select
+                   Given SQL SELECT query
+
+                Returns
+                -------
+                pd.DataFrame
+                    List of Football players matching the query
+
+                Raises
+                ------
+                ValueError
+                    If the query contains an unsupported condition
+                """
+        conditions = extract_comparison_conditions(query.where)
+        player_params = {}
+        for a_where in conditions:
+            key = a_where[1]
+
+            if a_where[0] != "=":
+                raise ValueError(f"Unsupported where operation for {key}")
+
+            # Assign values based on the key
+            if key == "league":
+                player_params["league"] = int(a_where[2])
+            elif key == "season":
+                player_params["season"] = int(a_where[2])
+            elif key == "team":
+                player_params["team"] = int(a_where[2])
+            elif key == "id":
+                player_params["id"] = int(a_where[2])
+            elif key == "search":
+                player_params["search"] = a_where[2]
+            elif key == "page":
+                player_params["page"] = int(a_where[2])
+            else:
+                raise ValueError(f"Unsupported where argument {key}")
+
+        client = self.handler.connect()
+        players_api = PlayersApi(client)
+        player_data = players_api.get_players(**player_params)
+
+        columns = []
+        for target in query.targets:
+            if isinstance(target, ast.Star):
+                columns = player_data.columns
+                break
+            elif isinstance(target, ast.Identifier):
+                columns.append(target.parts[-1])
+            else:
+                raise NotImplementedError
+
+        if len(player_data) == 0:
+            return pd.DataFrame([], columns=columns)
+
+        # Remove columns not part of select.
+        for col in set(player_data.columns).difference(set(columns)):
+            player_data = player_data.drop(col, axis=1)
+
+        return player_data
+
+    def get_columns(self) -> list:
+        """Gets all columns to be returned in pandas DataFrame responses"""
+
+        return [
+            "player_id",
+            "player_name",
+            "player_firstname",
+            "player_lastname",
+            "player_age",
+            "player_birth_date",
+            "player_birth_place",
+            "player_birth_country",
+            "player_nationality",
+            "player_height",
+            "player_weight",
+            "player_injured",
+            "player_photo",
+            "statistics_0_team_id",
```

Comment:
Is it possible to remove the `0` in these column names? This is not a big problem, but I was just wondering if it is possible because from what I could see the API only returns one record for each of these statistics?